src/ralph/parsers.py

```diff
@@ -58,12 +56,11 @@ def parse(self, input_file, chunksize=DEFAULT_GELF_PARSER_CHUNCK_SIZE):
         """
         logger.info("Parsing: %s", input_file)
 
-        if isinstance(input_file, str) and not Path(input_file).exists():
-            msg = "Input GELF log file '%s' does not exist"
-            logger.error(msg, input_file)
-            raise OSError(msg % (input_file))
-
-        chunks = pd.read_json(input_file, lines=True, chunksize=chunksize)
-        for chunk in chunks:
-            for event in chunk["short_message"].values:
-                yield event
+        for event in input_file:
+            try:
+                yield json.loads(event)["short_message"]
+            except (json.JSONDecodeError, TypeError, KeyError) as err:
+                logger.error(
+                    "Invalid event! Not JSON parsable or short_message missing!"
+                )
+                logger.debug("%s%s", type(err), err)
```

Comment:
```suggestion
            except (json.JSONDecodeError, TypeError) as err:
                logger.error(
                    "Input event '%s' is not a valid JSON string! It will be ignored.", 
                    event
                )
                logger.debug("Raised error was: %s", err)
            except KeyError as err:
                logger.error(
                    "Input event '%s' doesn't comply with GELF format! It will be ignored.", 
                    event
                )
                logger.debug("Raised error was: %s", err)
```